pacemaker.spec.in

```diff
@@ -630,6 +653,7 @@ exit 0
 
 %config(noreplace) %{_sysconfdir}/sysconfig/pacemaker
 %if %{defined _unitdir}
+%ghost %{_localstatedir}/lib/rpm-state/%{name}
```

Comment:
They would be removed as usual, when removing old package, totally or when installing (newer/older/same version) over.